package.json

```diff
@@ -19,7 +19,7 @@
     "node": ">=0.8"
   },
   "dependencies": {
-    "backbone": "1.1.2",
+    "backbone": "1.2.3",
```

Comment:
> We've always locked versions with Backbone since it doesn't follow semantic versioning.

https://github.com/tbranyen/backbone.layoutmanager/commit/ffe7afb7388f5249ff18ddf12788c10073cd5b47#diff-b9cfc7f2cdf78a7f4b91a753d10865a2